.github/src/content/publication/abz14/arcaini2014abz.md

```diff
@@ -85,12 +85,12 @@ pages="36--51",
 abstract="The paper presents an Abstract State Machine (ASM) specification of the Landing Gear System case study, and shows how the ASMETA framework can be used to support the modeling and analysis (validation and verification) activities for developing a rigorous and correct model in terms of ASMs. We exploit the two fundamental concepts of the ASM method, i.e., the notion of ground model and the refinement principle, and we achieve model development and model analysis by the combined use of formal methods for specification and for verification.",
 isbn="978-3-319-07512-9"
 }
-~~~
+```
```

Comment:
@nksingh-n7 both syntax styles are valid for a markdown source code block

Reply:
I think vscode had automatically generated this symbol for the markdown....